src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/ManageEvidenceNotesControllerDownloadEvidenceNotesReportTests.cs

```diff
@@ -97,5 +87,41 @@ public async Task DownloadEvidenceNoteReportGet_GivenCsvData_FileContentResultSh
             result.FileDownloadName.Should().Be(CsvFilenameFormat.FormatFileName(csvFile.FileName));
             result.FileDownloadName.Should().Be(csvFile.FileName);
         }
+
+        [Fact]
+        public void DownloadEvidenceNotePdfGet_ShouldHaveHttpGetAttribute()
+        {
+            typeof(ManageEvidenceNotesController).GetMethod("DownloadEvidenceNotePdf", new[] { typeof(Guid) }).Should()
+                .BeDecoratedWith<HttpGetAttribute>();
+        }
+
+        [Fact]
+        public async Task DownloadEvidenceNotePdfGet_GivenPdf_FileShouldBeReturned()
+        {
+            //arrange
+            SystemTime.Freeze(new DateTime(2022, 10, 5, 13, 22, 10));
+            var pdf = Fixture.Create<byte[]>();
+
+            var data = Fixture.Build<EvidenceNoteData>()
+                .With(e => e.ComplianceYear, 2022)
+                .Create();
+
+            var model = Fixture.Build<ViewEvidenceNoteViewModel>()
+                .With(v => v.Reference, 1200)
+                .With(v => v.Type, NoteType.Evidence).Create();
+
+            A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>._)).Returns(data);
```

Comment:
@GuidoWorkDev If you are going to cover the entire action in a single test these setups of A.CallTo should test for exact values.

A.CallTo(() => WeeeClient.SendAsync(A<string>._, A<GetEvidenceNoteForSchemeRequest>.That.Matches.....)).Returns(data); etc

For all the setups...